frontend/src/components/widgets/list-view/ListView.jsx

```diff
@@ -157,7 +166,9 @@ function ListView({
               title={renderTitle(item)}
               description={
                 <Typography.Text type="secondary" ellipsis>
```

Comment:
@Deepak-Kesavan NIT: Were you able to test the ellipsis? I think only if we set the width, the ellipsis will work. Can you please verify this?

Also, we can add something like this to add tooltip to the `Typography.Text` component - `ellipsis={{ tooltip: 'I am ellipsis now!' }}`. Please refer the [doc](https://ant.design/components/typography#components-typography-demo-ellipsis) for more context.